src/app/client/src/app/modules/telemetry/services/telemetry/telemetry.service.ts

```diff
@@ -86,8 +86,9 @@ export class TelemetryService {
    * @param {IStartEventInput} startEventInput
    * @memberof TelemetryService
    */
-  public start(startEventInput: IStartEventInput) {
+  public async start(startEventInput: IStartEventInput) {
```

Comment:
why is this async function

Reply:
because I am making modification in each and every data formation call so waiting for the data to format well.